internal/v4/search_test.go

```diff
@@ -43,12 +51,42 @@ var exportTestBundles = map[string]string{
 func (s *SearchSuite) SetUpTest(c *gc.C) {
 	s.IsolatedMgoESSuite.SetUpTest(c)
 	si := &charmstore.SearchIndex{s.ES, s.TestIndex}
-	s.srv, s.store = newServer(c, s.Session, si, serverParams)
+	var mu sync.Mutex
+	s.discharger = bakerytest.NewDischarger(nil, func(cond string, arg string) ([]checkers.Caveat, error) {
```

Comment:
I don't think we need to do all this. We could just mint a macaroon with no 3rd party caveats that has the required declared caveats. No need to start a discharge service, I think.